--- opus-1.1/configure.ac.orig	2013-12-05 00:26:45.000000000 +0100
+++ opus-1.1/configure.ac	2013-12-06 18:09:55.758855450 +0100
@@ -333,9 +333,9 @@
 
 AM_CONDITIONAL([CPU_ARM], [test "$cpu_arm" = "yes"])
 AM_CONDITIONAL([OPUS_ARM_INLINE_ASM],
-    [test x"${inline_optimization:0:3}" = x"ARM"])
+    [echo x"${inline_optimization}" | grep -q "^xARM"])
 AM_CONDITIONAL([OPUS_ARM_EXTERNAL_ASM],
-    [test x"${asm_optimization:0:3}" = x"ARM"])
+    [echo x"${asm_optimization}" | grep -q "^xARM"])
 
 AS_IF([test x"$enable_rtcd" = x"yes"],[
     AS_IF([test x"$rtcd_support" != x"no"],[
